include/caffe/loss_layers.hpp

```diff
@@ -691,14 +691,7 @@ template <typename Dtype> class SoftmaxLayer;
 template <typename Dtype>
 class SoftmaxWithLossLayer : public LossLayer<Dtype> {
  public:
-   /**
-    * @param param provides LossParameter loss_param, with options:
-    *  - ignore_label (optional)
-    *    Specify a label value that should be ignored when computing the loss.
-    *  - normalize (optional, default true)
-    *    If true, the loss is normalized by the number of (nonignored) labels
-    *    present; otherwise the loss is simply summed over spatial locations.
-    */
```

Comment:
You're right, that documentation is correct. I was misreading it, sorry. It's actually the ``@copydoc SoftmaxWithLossLayer`` in Forward_cpu that triggers the doxygen warning.

I can update the PR with the doc for Forward_cpu removed as I don't think the same docs make sense in Forward_cpu.